addons/rose/addon/styles/hds/themes/dark-mode/_color-tokens-tier-2.scss

```diff
@@ -0,0 +1,162 @@
+  --token-color-focus-critical-internal: var(--token-color-palette-red-300);
+  // --token-color-focus-critical-external: #dd7578; /* this is a special color used only for the focus style, to pass color contrast for a11y purpose */
+  --token-color-foreground-strong: var(--token-color-palette-neutral-700);
+  --token-color-foreground-primary: var(--token-color-palette-neutral-600);
+  --token-color-foreground-faint: var(--token-color-palette-neutral-500);
+  --token-color-foreground-high-contrast: var(--token-color-palette-neutral-0);
+  --token-color-foreground-disabled: var(--token-color-palette-neutral-400);
+  --token-color-foreground-action: var(--token-color-palette-blue-200);
+  --token-color-foreground-action-hover: var(--token-color-palette-blue-300);
+  --token-color-foreground-action-active: var(--token-color-palette-blue-400);
+  --token-color-foreground-highlight: var(--token-color-palette-purple-200);
+  --token-color-foreground-highlight-on-surface: var(--token-color-palette-purple-300);
+  --token-color-foreground-highlight-high-contrast: var(--token-color-palette-purple-500);
+  --token-color-foreground-success: var(--token-color-palette-green-200);
+  --token-color-foreground-success-on-surface: var(--token-color-palette-green-300);
+  --token-color-foreground-success-high-contrast: var(--token-color-palette-green-500);
+  --token-color-foreground-warning: var(--token-color-palette-amber-200);
+  --token-color-foreground-warning-on-surface: var(--token-color-palette-amber-300);
+  --token-color-foreground-warning-high-contrast: var(--token-color-palette-amber-500);
+  --token-color-foreground-critical: var(--token-color-palette-red-200);
+  --token-color-foreground-critical-on-surface: var(--token-color-palette-red-300);
+  --token-color-foreground-critical-high-contrast: var(--token-color-palette-red-500);
+  --token-color-page-primary: var(--token-color-palette-neutral-0);
+  --token-color-page-faint: var(--token-color-palette-neutral-50);
+  --token-color-surface-primary: var(--token-color-palette-neutral-0);
+  --token-color-surface-faint: var(--token-color-palette-neutral-50);
+  --token-color-surface-strong: var(--token-color-palette-neutral-100);
+  --token-color-surface-interactive: var(--token-color-palette-neutral-0);
+  --token-color-surface-interactive-hover: var(--token-color-palette-neutral-100);
+  --token-color-surface-interactive-active: var(--token-color-palette-neutral-200);
+  --token-color-surface-interactive-disabled: var(--token-color-palette-neutral-50);
+  --token-color-surface-action: var(--token-color-palette-blue-50);
+  --token-color-surface-highlight: var(--token-color-palette-purple-50);
+  --token-color-surface-success: var(--token-color-palette-green-50);
+  --token-color-surface-warning: var(--token-color-palette-amber-50);
+  --token-color-surface-critical: var(--token-color-palette-red-50);
+
+  // --token-color-hashicorp-brand: #000000;
```

Comment:
What are all these commented tokens for?

Reply:
They're for reference.  They appear as "first tier" tokens in HDS, but we aren't mapping them yet.